actors/miner/src/lib.rs

```diff
@@ -3757,8 +3758,12 @@ fn extend_simple_qap_sector(
     claim_space_by_sector: &BTreeMap<SectorNumber, (u64, u64)>,
 ) -> Result<SectorOnChainInfo, ActorError> {
     let mut new_sector = sector.clone();
+    // Update the power_base_epoch, assuming the sector is actually being extended
```

Comment:
Note that I am NOT setting the power_base_epoch for the `extend_non_simple_qap_sector` method below (see [here](https://github.com/filecoin-project/builtin-actors/issues/914#issuecomment-1464566239) for more).